frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -380,7 +380,7 @@ button.usa-pagination__button.usa-button {
 }
 
 @include at-media("desktop") {
-  .desktop\:margin-bottom-5px {
-    margin-bottom: 5px !important;
+  .desktop\:margin-bottom-3px {
```

Comment:
USWDS has margins of 1px, 2px, or 4px:

![image](https://github.com/user-attachments/assets/d8dbc120-42db-4fd9-9589-f28bb49927e2)